Sources/StrongContractResponse/String.swift

```diff
@@ -17,6 +17,7 @@ extension String {
 
         guard let url = defaultPath ??
                 typicalKeys.compactMap({ usualCandidates(for: $0) }).first else {
```

Comment:
This can also be written without a closure. You can pass the initializer directly. try 
`compactMap(usualCandidates.init)`

Reply:
If there multiples methods with a simple parameter with the same type you can force the one you want

`usualCandidates(for:)`

Reply:
pushed a commit for that.